src/index.ts

```diff
@@ -49,6 +49,14 @@ export { UnmatchedRoute } from "./interfaces/unmatched-route";
 
 // #endregion Interfaces
 
+// -----------------------------------------------------------------------------------------
+// #region Reducers
```

Comment:
Hmm - while the core of this hook is a reducer, I wonder if we should keep it in the `hooks` folder & region for now until we start to have multiple reducer actors?

Reply:
Now that I'm thinking about it, I think we could actually consider this a hook anyway. It's just a hook that uses a reducer.